exercise_3/src/IRecord.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <string>
+
+class IRecord {
+public:
+	int _id;
```

Comment:
This is not good approach - rather avoid creation of public members. Please move it to "private" (and create setter and getter) or "protected".

Reply:
Changed to "protected".